### ISSUE TITLE
[Fluxstore] -Unable to Build for ios 

### ISSUE THREAD

**Issue by tjain90** [inspireui/support#2623]
> - Thank you for reaching please follow the format below to create a new ticket, please explain as more detail as possible and allow 24 - 48 hours to receive an answer to your ticket.
> - Try to explain your all problems in one ticket, instead of creating multi tickets.
> - After creating an issue, please take the screenshot of http://codecanyon.net/download - to let us know you have purchased our product, or quick note as a comment that you have created this ticket, example: https://tppr.me/DJA8L
> - IMPORTANT: We will close the ticket if you could not prove as our customer.

---------

1. My issue: Unable to buildfor IOS.Throwing error for razor pay and several others.

2. Screenshots issues (required):

3. Proof of my purchased (required): tjain90@gmail.com from themeforest

3. Config files (required)

Here is my log:

Launching lib/main.dart on iPhone 11 Pro Max in debug mode...
log: Must be admin to run 'stream' command
Script started, output file is /dev/null

Script done, output file is /dev/null
Running pod install...
Running Xcode build...
Xcode build done.                                           47.3s
Failed to build iOS app
Error output from Xcode build:
↳
    2020-04-08 23:52:25.804 xcodebuild[64185:1492989] Malformed bundle does not contain an identifier at /Library/Developer/CoreSimulator/Profiles/Runtimes/watchOS 4.1.simruntime
    2020-04-08 23:52:25.806 xcodebuild[64185:1492989] Malformed bundle does not contain an identifier at /Library/Developer/CoreSimulator/Profiles/Runtimes/tvOS 11.1.simruntime
    2020-04-08 23:52:25.806 xcodebuild[64185:1492989] Malformed bundle does not contain an identifier at /Library/Developer/CoreSimulator/Profiles/Runtimes/watchOS 4.0.simruntime
    2020-04-08 23:52:25.813 xcodebuild[64185:1492989] Malformed bundle does not contain an identifier at /Library/Developer/CoreSimulator/Profiles/Runtimes/tvOS 11.0.simruntime
    ** BUILD FAILED **

Xcode's output:
↳
    In file included from /Users/user167955/.pub-cache/hosted/pub.dartlang.org/uni_links-0.2.0/ios/Classes/UniLinksPlugin.m:1:
    /Users/user167955/.pub-cache/hosted/pub.dartlang.org/uni_links-0.2.0/ios/Classes/UniLinksPlugin.h:4:4: warning: pointer is missing a nullability type specifier (_Nonnull, _Nullable, or _Null_unspecified) [-Wnullability-completeness]
    + (instancetype)sharedInstance;
       ^
    /Users/user167955/.pub-cache/hosted/pub.dartlang.org/uni_links-0.2.0/ios/Classes/UniLinksPlugin.h:4:4: note: insert '_Nullable' if the pointer may be null
    + (instancetype)sharedInstance;
       ^
                    _Nullable 
    /Users/user167955/.pub-cache/hosted/pub.dartlang.org/uni_links-0.2.0/ios/Classes/UniLinksPlugin.h:4:4: note: insert '_Nonnull' if the pointer should never be null
    + (instancetype)sharedInstance;
       ^
                    _Nonnull 
    /Users/user167955/.pub-cache/hosted/pub.dartlang.org/uni_links-0.2.0/ios/Classes/UniLinksPlugin.h:7:33: warning: block pointer is missing a nullability type specifier (_Nonnull, _Nullable, or _Null_unspecified) [-Wnullability-completeness]
          restorationHandler:(void (^)(NSArray *_Nullable))restorationHandler;
                                    ^
    /Users/user167955/.pub-cache/hosted/pub.dartlang.org/uni_links-0.2.0/ios/Classes/UniLinksPlugin.h:7:33: note: insert '_Nullable' if the block pointer may be null
          restorationHandler:(void (^)(NSArray *_Nullable))restorationHandler;
                                    ^
                                     _Nullable
    /Users/user167955/.pub-cache/hosted/pub.dartlang.org/uni_links-0.2.0/ios/Classes/UniLinksPlugin.h:7:33: note: insert '_Nonnull' if the block pointer should never be null
          restorationHandler:(void (^)(NSArray *_Nullable))restorationHandler;
                                    ^
                                     _Nonnull
    2 warnings generated.
    /Users/user167955/.pub-cache/hosted/pub.dartlang.org/razorpay_flutter-1.1.2/ios/Classes/RazorpayDelegate.swift:57:24: error: module 'Razorpay' has no member named 'initWithKey'
            let razorpay = Razorpay.initWithKey(key ?? "", andDelegateWithData: self)
                           ^~~~~~~~ ~~~~~~~~~~~
    /Users/user167955/.pub-cache/hosted/pub.dartlang.org/rate_my_app-0.6.0+2/ios/Classes/RateMyAppPlugin.m:15:30: warning: 'requestReview' is only available on iOS 10.3 or newer [-Wunguarded-availability]
        [SKStoreReviewController requestReview];
                                 ^~~~~~~~~~~~~~
    In module 'StoreKit' imported from /Users/user167955/.pub-cache/hosted/pub.dartlang.org/rate_my_app-0.6.0+2/ios/Classes/RateMyAppPlugin.m:2:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.4.sdk/System/Library/Frameworks/StoreKit.framework/Headers/SKStoreReviewController.h:22:1: note: 'requestReview' has been marked as being introduced in iOS 10.3 here, but the deployment target is iOS 8.0.0
    + (void)requestReview API_AVAILABLE(ios(10.3), macos(10.14)) API_UNAVAILABLE(watchos) __TVOS_PROHIBITED;
    ^
    /Users/user167955/.pub-cache/hosted/pub.dartlang.org/rate_my_app-0.6.0+2/ios/Classes/RateMyAppPlugin.m:15:30: note: enclose 'requestReview' in an @available check to silence this warning
        [SKStoreReviewController requestReview];
                                 ^~~~~~~~~~~~~~
    /Users/user167955/.pub-cache/hosted/pub.dartlang.org/rate_my_app-0.6.0+2/ios/Classes/RateMyAppPlugin.m:15:6: warning: 'SKStoreReviewController' is only available on iOS 10.3 or newer [-Wunguarded-availability]
        [SKStoreReviewController requestReview];
         ^~~~~~~~~~~~~~~~~~~~~~~
    In module 'StoreKit' imported from /Users/user167955/.pub-cache/hosted/pub.dartlang.org/rate_my_app-0.6.0+2/ios/Classes/RateMyAppPlugin.m:2:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.4.sdk/System/Library/Frameworks/StoreKit.framework/Headers/SKStoreReviewController.h:14:110: note: 'SKStoreReviewController' has been marked as being introduced in iOS 10.3 here, but the deployment target is iOS 8.0.0
    SK_EXTERN_CLASS API_AVAILABLE(ios(10.3), macos(10.14)) API_UNAVAILABLE(watchos) __TVOS_PROHIBITED @interface SKStoreReviewController : NSObject
                                                                                                                 ^
    /Users/user167955/.pub-cache/hosted/pub.dartlang.org/rate_my_app-0.6.0+2/ios/Classes/RateMyAppPlugin.m:15:6: note: enclose 'SKStoreReviewController' in an @available check to silence this warning
        [SKStoreReviewController requestReview];
         ^~~~~~~~~~~~~~~~~~~~~~~
    2 warnings generated.
    Command CompileSwift failed with a nonzero exit code
    note: Using new build system
    note: Building targets in parallel
    note: Planning build
    note: Constructing build description
    warning: Mapping architecture armv7 to i386. Ensure that this target's Architectures and Valid Architectures build settings are configured correctly for the iOS Simulator platform. (in target 'image_picker' from project 'Pods')
    warning: Mapping architecture arm64 to x86_64. Ensure that this target's Architectures and Valid Architectures build settings are configured correctly for the iOS Simulator platform. (in target 'image_picker' from project 'Pods')

**Comment by minhcasi** [comment]
@tjain90 - pls help to check this video guide https://www.youtube.com/watch?v=ReAUZDIgh2I

And clean the cache - https://gist.github.com/minhcasi/2362b8ed369738cea2bf10a57ac569e1

**Comment by tjain90** [comment]
Thanks for this. It sorted out most of the issue but still unable to build with below error:

Error output from Xcode build:
↳
    ** BUILD FAILED **


Xcode's output:
↳
    /Users/user167955/.pub-cache/hosted/pub.dartlang.org/razorpay_flutter-1.1.2/ios/Classes/RazorpayDelegate.swift:57:24: error: module 'Razorpay' has no member named 'initWithKey'
            let razorpay = Razorpay.initWithKey(key ?? "", andDelegateWithData: self)
                           ^~~~~~~~ ~~~~~~~~~~~
    Command CompileSwift failed with a nonzero exit code
    note: Using new build system
    note: Building targets in parallel
    note: Planning build
    note: Constructing build description
    warning: Mapping architecture armv7 to i386. Ensure that this target's Architectures and Valid Architectures build settings are configured correctly for the iOS Simulator platform. (in target 'image_picker' from project 'Pods')
    warning: Mapping architecture arm64 to x86_64. Ensure that this target's Architectures and Valid Architectures build settings are configured correctly for the iOS Simulator platform. (in target 'image_picker' from project 'Pods')

Could not build the application for the simulator.
Error launching application on iPhone 11 Pro Max.

**Comment by tjain90** [comment]
I have replaced ios directory from new version of code but still throws lot of error:

Script done, output file is /dev/null

Running pod install...
Running Xcode build...
Xcode build done.                                           73.7s
Failed to build iOS app
Error output from Xcode build:
↳
    ** BUILD FAILED **


Xcode's output:
↳
    In file included from /Users/user167955/.pub-cache/hosted/pub.dartlang.org/uni_links-0.2.0/ios/Classes/UniLinksPlugin.m:1:
    /Users/user167955/.pub-cache/hosted/pub.dartlang.org/uni_links-0.2.0/ios/Classes/UniLinksPlugin.h:4:4: warning: pointer is missing a nullability type specifier (_Nonnull, _Nullable, or _Null_unspecified) [-Wnullability-completeness]
    + (instancetype)sharedInstance;
       ^
    /Users/user167955/.pub-cache/hosted/pub.dartlang.org/uni_links-0.2.0/ios/Classes/UniLinksPlugin.h:4:4: note: insert '_Nullable' if the pointer may be null
    + (instancetype)sharedInstance;
       ^
                    _Nullable 
    /Users/user167955/.pub-cache/hosted/pub.dartlang.org/uni_links-0.2.0/ios/Classes/UniLinksPlugin.h:4:4: note: insert '_Nonnull' if the pointer should never be null
    + (instancetype)sharedInstance;
       ^
                    _Nonnull 
    /Users/user167955/.pub-cache/hosted/pub.dartlang.org/uni_links-0.2.0/ios/Classes/UniLinksPlugin.h:7:33: warning: block pointer is missing a nullability type specifier (_Nonnull, _Nullable, or _Null_unspecified) [-Wnullability-completeness]
          restorationHandler:(void (^)(NSArray *_Nullable))restorationHandler;
                                    ^
    /Users/user167955/.pub-cache/hosted/pub.dartlang.org/uni_links-0.2.0/ios/Classes/UniLinksPlugin.h:7:33: note: insert '_Nullable' if the block pointer may be null
          restorationHandler:(void (^)(NSArray *_Nullable))restorationHandler;
                                    ^
                                     _Nullable
    /Users/user167955/.pub-cache/hosted/pub.dartlang.org/uni_links-0.2.0/ios/Classes/UniLinksPlugin.h:7:33: note: insert '_Nonnull' if the block pointer should never be null
          restorationHandler:(void (^)(NSArray *_Nullable))restorationHandler;
                                    ^
                                     _Nonnull
    2 warnings generated.
    /Users/user167955/.pub-cache/hosted/pub.dartlang.org/razorpay_flutter-1.1.2/ios/Classes/RazorpayDelegate.swift:57:24: error: module 'Razorpay' has no member named 'initWithKey'
            let razorpay = Razorpay.initWithKey(key ?? "", andDelegateWithData: self)
                           ^~~~~~~~ ~~~~~~~~~~~
    /Users/user167955/.pub-cache/hosted/pub.dartlang.org/rate_my_app-0.6.0+2/ios/Classes/RateMyAppPlugin.m:15:30: warning: 'requestReview' is only available on iOS 10.3 or newer [-Wunguarded-availability]
        [SKStoreReviewController requestReview];
                                 ^~~~~~~~~~~~~~
    In module 'StoreKit' imported from /Users/user167955/.pub-cache/hosted/pub.dartlang.org/rate_my_app-0.6.0+2/ios/Classes/RateMyAppPlugin.m:2:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.4.sdk/System/Library/Frameworks/StoreKit.framework/Headers/SKStoreReviewController.h:22:1: note: 'requestReview' has been marked as being introduced in iOS 10.3 here, but the deployment target is iOS 8.0.0
    + (void)requestReview API_AVAILABLE(ios(10.3), macos(10.14)) API_UNAVAILABLE(watchos) __TVOS_PROHIBITED;
    ^
    /Users/user167955/.pub-cache/hosted/pub.dartlang.org/rate_my_app-0.6.0+2/ios/Classes/RateMyAppPlugin.m:15:30: note: enclose 'requestReview' in an @available check to silence this warning
        [SKStoreReviewController requestReview];
                                 ^~~~~~~~~~~~~~
    /Users/user167955/.pub-cache/hosted/pub.dartlang.org/rate_my_app-0.6.0+2/ios/Classes/RateMyAppPlugin.m:15:6: warning: 'SKStoreReviewController' is only available on iOS 10.3 or newer [-Wunguarded-availability]
        [SKStoreReviewController requestReview];
         ^~~~~~~~~~~~~~~~~~~~~~~
    In module 'StoreKit' imported from /Users/user167955/.pub-cache/hosted/pub.dartlang.org/rate_my_app-0.6.0+2/ios/Classes/RateMyAppPlugin.m:2:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.4.sdk/System/Library/Frameworks/StoreKit.framework/Headers/SKStoreReviewController.h:14:110: note: 'SKStoreReviewController' has been marked as being introduced in iOS 10.3 here, but the deployment target is iOS 8.0.0
    SK_EXTERN_CLASS API_AVAILABLE(ios(10.3), macos(10.14)) API_UNAVAILABLE(watchos) __TVOS_PROHIBITED @interface SKStoreReviewController : NSObject
                                                                                                                 ^
    /Users/user167955/.pub-cache/hosted/pub.dartlang.org/rate_my_app-0.6.0+2/ios/Classes/RateMyAppPlugin.m:15:6: note: enclose 'SKStoreReviewController' in an @available check to silence this warning
        [SKStoreReviewController requestReview];
         ^~~~~~~~~~~~~~~~~~~~~~~
    2 warnings generated.
    Command CompileSwift failed with a nonzero exit code
    note: Using new build system
    note: Building targets in parallel
    note: Planning build
    note: Constructing build description
    warning: Mapping architecture armv7 to i386. Ensure that this target's Architectures and Valid Architectures build settings are configured correctly for the iOS Simulator platform. (in target 'image_picker' from project 'Pods')
    warning: Mapping architecture arm64 to x86_64. Ensure that this target's Architectures and Valid Architectures build settings are configured correctly for the iOS Simulator platform. (in target 'image_picker' from project 'Pods')

Could not build the application for the simulator.
Error launching application on iPhone 11 Pro Max.

**Comment by minhcasi** [comment]
@tjain90 - could you try to run the project at stable channel
```
flutter channel stable
flutter upgrade --force
```

**Comment by tjain90** [comment]
Hi already run both commands. Flutter is at stable channel only.

**Comment by minhcasi** [comment]
@tjain90 - pls try to clean cache by using this guide - https://docs.inspireui.com/fluxstore#fluxstore-multi-vendor-app

I have also created the video on this link https://www.youtube.com/watch?v=ReAUZDIgh2I

**Comment by vanlalruata** [comment]
> 
> 
> Thanks for this. It sorted out most of the issue but still unable to build with below error:
> 
> Error output from Xcode build:
> ↳
> ** BUILD FAILED **
> 
> Xcode's output:
> ↳
> /Users/user167955/.pub-cache/hosted/pub.dartlang.org/razorpay_flutter-1.1.2/ios/Classes/RazorpayDelegate.swift:57:24: error: module 'Razorpay' has no member named 'initWithKey'
> let razorpay = Razorpay.initWithKey(key ?? "", andDelegateWithData: self)
> ^~~~~~~~ ~~~~~~~~~~~
> Command CompileSwift failed with a nonzero exit code
> note: Using new build system
> note: Building targets in parallel
> note: Planning build
> note: Constructing build description
> warning: Mapping architecture armv7 to i386. Ensure that this target's Architectures and Valid Architectures build settings are configured correctly for the iOS Simulator platform. (in target 'image_picker' from project 'Pods')
> warning: Mapping architecture arm64 to x86_64. Ensure that this target's Architectures and Valid Architectures build settings are configured correctly for the iOS Simulator platform. (in target 'image_picker' from project 'Pods')
> 
> Could not build the application for the simulator.
> Error launching application on iPhone 11 Pro Max.

**I think alternative payment method like GPay will be nice.**

RazorPay is not currently active in native. But only webview. :-|

**Comment by minhcasi** [comment]
@vanlalruata - you can try to use the latest version 1.6.6b that already fix this build issue - caused by the Razopay library

**Comment by tjain90** [comment]
Yes Updating app module fix this but i am unable to build. it throws an error: "doesn't include signing certificate "Apple Distribution"

I have created distribution and development certificate in apple development portal. also created profile and imported to xcode project. Still it throws an signing error.
